lib/Dialect/Torch/Utils/Utils.cpp

```diff
@@ -36,6 +36,18 @@ Torch::matchLegalConstantIndexIntoListOfSize(Value v, int64_t length) {
   return dim;
 }
 
+Value Torch::toIntListConstruct(PatternRewriter &rewriter, Location loc,
+                                ArrayRef<int64_t> cstInput,
+                                Torch::IntType intType) {
```

Comment:
There is only one torch int type, so this should not be a function argument.

Reply:
Changed the signature in latest commit